src/context/status.ts

```diff
@@ -1,4 +1,4 @@
-import statuses from 'statuses/codes.json'
+import statuses from 'statuses/codes.json' assert { type: 'json' }
```

Comment:
this will currently be removed on build

Reply:
we could also configure `tsup` to inline this library by moving it to `devDependencies` instead of listing in dependencies